hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -379,6 +379,9 @@ private MemStore getMemstore() {
   protected void createCacheConf(final ColumnFamilyDescriptor family) {
     this.cacheConf = new CacheConfig(conf, family, region.getBlockCache(),
         region.getRegionServicesForStores().getByteBuffAllocator());
+    LOG.info("Created cacheConfig: " + this.getCacheConfig()
```

Comment:
Looks good. There is another jira where we discuss abt  changing HStore#toString() to add region info also there. Then u will need to change this also.

Reply:
> Looks good. There is another jira where we discuss abt changing HStore#toString() to add region info also there. Then u will need to change this also.

But, HStore#toString() print only short name of column family with out any attributes, I am not sure is it suitable to use here, what is your suggestion?

Reply:
Ya that should be fine IMO